native/packages/react-native-bpk-component-text-input/readme.md

```diff
@@ -41,6 +41,7 @@ export default class App extends Component {
           label="City"
           style={styles.input}
           value="Edinburgh"
+          description="What's the capital of Scotland?"
```

Comment:
Is this really how we want the `description` prop to be used?

Reply:
No, I was being whimsical but I guess it's better to lead by example and show a useful message! I'll change it.